src/types/select.js

```diff
@@ -4,12 +4,42 @@ import assign from "lodash.assign";
 import css from "./select.css";
 import multiple from "./lib/multiple.js";
 
+function findSelected(children) {
+    return children.find((option) => Boolean(option.selected));
```

Comment:
I guess not. I just have a habit of casting to Boolean whenever that's what I'm actually looking for, mostly out of instinct and regardless of whether it's specifically necessary. It's often not necessary, but it feels safer. I've used `!!` for it in the past, but I know you hate that, haha. 

In any case, I can remove it if you'd like. No biggie.